.github/workflows/rust.yml

```diff
@@ -64,3 +64,16 @@ jobs:
       env:
         CARGO_TARGET_X86_64_UNKNOWN_LINUX_GNU_RUNNER: "valgrind --leak-check=full"
       run: cargo test --release --features alloc
+
+  miri:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: dtolnay/rust-toolchain@nightly
+        with: 
+          components: miri
+      - run: |
+          cargo miri test
```

Comment:
run tests only with `feature alloc`
there is no point in running without it

Reply:
Done